hydrophone.go

```diff
@@ -1,3 +1,20 @@
+// @title Hydrophone API
+// @version 0.0.1
```

Comment:
Should we use the v0.0.1 for the API description ?

Reply:
Good question, to discuss with the team